templates/security/cve/_cve-table.html

```diff
@@ -98,6 +98,40 @@
                   {% endif %}
                 </small>
               </div>
+              <div class="icon-container__icon icon-cve-info">
+                {% if statuses[release.codename].status != 'DNE'%}
+                  <i class="p-icon--information"></i>
+                {% endif %}
+                <small class="cve-tooltip">
```

Comment:
If I read this correctly, it includes the popup text inside every cell. That seems like it would multiply the weight of the page, especially long search results pages (and be tiresome for text-mode browsers).

Have you considered instead generating the popup with script when the icon is clicked? Perhaps something like:
- the icon has `<a href="#key">`
- `<dl id="key">`, after the table, contains `<dt>` `<dd>` with the text for each status and priority
- if you have JavaScript, you get (a) `#key {display: none}`, because instead you have (b) a script that overrides all the `href="#key"` with popup-creating functions.

Reply:
The pop-up is indeed generated in every cell and with the help of CSS it get's displayed on hover. 
This way I avoid using javascript which I thought was the cleaner/optimal way to get this working. Also the tool-top is just plain text element. 

However I do not mind changing it if dynamically generating the content of the tool-tip on hover of the icon is the better solution. What do you think, @steverydz?

Reply:
I think what @matthewpaulthomas is saying is right. It would be nice to do this without JS but having the text in every cell could cause problems with search, accessibility etc, so I think doing it dynamically might be the better solution. It does mean there will be no tooltip without JS but that's probably not a major concern in this case.